theme/less/_content.less

```diff
@@ -2,3 +2,31 @@ main article img
 {
   max-width: 100%;
 }
+
+article
+{
+  color: #444444;
+}
+
+article p, article ul, article ol
```

Comment:
In the Drupal version of the website `ul` and `ol` have top and bottom margins, I wonder if we need those here as well?

Reply:
What you think is it  OK for these elements to have same font sizes and line-heights?

Reply:
For line-height to have same height as font-size `line-height: 1` can be used AFAIK. In general that's a UX question, so I would say rely on your own, for some fonts having `line-height: 1` specified can make lines very close to each other affecting readability.